atlp-rwanda/wolves-bn-backend
# Added the eslint file and the hound.yml file for houndCI

#### What does this PR do?
- Add Eslint and HoundCI file

#### How should this be manually tested?
-Try to declare a variable and don't use it. You should be able to see the linting problem in the terminal.

#### Any background context you want to provide?
Added the Eslint and HoundCI to check any codes that are formatted badly or some bad practice.

#### What are the relevant pivotal tracker stories?
#174772158
